module/project/templateTasks/helper.js

```diff
@@ -112,32 +110,30 @@ module.exports = class ProjectTemplateTasksHelper {
                 // }
 
                 let solutionData = {};
+                // console.log("-------------------------------------------solutionIds",solutionIds)
+                if ( solutionIds.length > 0 ) {
+                    let solutions = await database.models.solutions.find({ "_id": { "$in": solutionIds } })
```

Comment:
@Prajwal17Tunerlabs please don't fetch from the model directly add solution in databaseQueries and add the function there.